pkg/runner_manager/manager.go

```diff
@@ -24,6 +26,10 @@ import (
 	"github.com/tum-dev/gocast/runner/protobuf"
 )
 
+var logger = log.New(log.NewJSONHandler(os.Stdout, &log.HandlerOptions{
+	Level: log.LevelDebug,
+})).With("service", "runner_manager")
```

Comment:
Ideally, logger should not be package global, but a member of `*Manager` and initialized in `runner_manager.New`[1]. Also, please apply it to all logging in this file if you introduce a new logging approach, e.g. here https://github.com/TUM-Dev/gocast/blob/515502a3c9543ba42195b52537f90fb3bf3e54c3/pkg/runner_manager/manager.go#L61

[1]: https://github.com/TUM-Dev/gocast/blob/515502a3c9543ba42195b52537f90fb3bf3e54c3/pkg/runner_manager/manager.go#L42